repository/src/test/java/org/apache/atlas/repository/util/FilterUtilTest.java

```diff
@@ -0,0 +1,62 @@
+package org.apache.atlas.repository.util;
+
+import org.junit.Test;
+
+import static org.apache.atlas.repository.util.FilterUtil.validateFilePath;
+import static org.junit.Assert.*;
+
+public class FilterUtilTest {
+    @Test
+    public void testValidateFilePath_ValidPath() {
```

Comment:
See if possible to only club all assers into a single Test method

Reply:
I can i guess but why club them, can you explain why?

Reply:
IMO all we do is calling a method to validate different strings, i think one method is enough for that & also error message that we pass in asset, already have description of test

Reply:
Atlas repo already has some test classes where we follow the same pattern, as a sample check DSLQueriesTest.comparison. it has a list of queries on what it performs exactly the same checks without multiple methods

Reply:
Alright, will do that